ui/tests/integration/components/variable-paths-test.js

```diff
@@ -77,21 +77,21 @@ module('Integration | Component | variable-paths', function (hooks) {
       .dom('tbody tr:first-child td:first-child a')
       .hasAttribute(
         'href',
-        '/ui/variables/foo/bar/baz',
+        '/ui/variables/var/foo/bar/baz',
```

Comment:
Side-effects: correcting for a previous change to add /var to disambiguate variable path URLs